programs/loader-v4/src/lib.rs

```diff
@@ -1553,4 +1553,32 @@ mod tests {
             Err(InstructionError::InvalidAccountData),
         );
     }
+
+    #[test]
+    fn test_small_header() {
+        let program_address = Pubkey::new_unique();
+        let transaction_accounts = vec![
+            (
+                program_address,
+                AccountSharedData::new(
+                    1,
+                    LoaderV4State::program_data_offset() - 1,
+                    &loader_v4::id(),
+                ),
+            ),
+            (
+                Pubkey::new_unique(),
```

Comment:
Is this supposed to be the authority? Why have it being owned by the program, and why so many lamports?